src/102610_len_2330.c

```diff
@@ -83,16 +78,13 @@ void func_802E12F8(void) {
 }
 
 void func_802E1328(Entity* entity) {
-    s32* temp = &D_802EB394;
-
-    if (*temp != 0) {
+    if (D_802EB394 != 0) {
         exec_entity_updatecmd(entity);
     }
 }
 
 void func_802E1350(Entity* entity) {
-    s32* temp = &D_802EB390;
-    if (*temp != 0) {
+    if (D_802EB390 != 0) {
```

Comment:
This and `D_802EB394` are bools, so we can drop the `!= 0`.

Reply:
Also elsewhere -- e.g. `D_802EB390 = 0` up above -- can be changed to using bool defs too